packages/sdk/src/types/runtime/actor.ts

```diff
@@ -761,18 +752,18 @@ export class Actor implements ActorLike, Patchable<ActorLike> {
 	}
 
 	private _setCollider(collider: Partial<ColliderLike>) {
-		if (this._collider) {
-			unobserve(this._collider);
-			this._collider = undefined;

```

Comment:
Why is this being removed?  We need to unobserve from the previous collider if there is one.